Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+
+RUN yum update -y
+RUN yum install -y unzip
+LABEL ORG="Armedia LLC" \
```

Comment:
We need to use `ArkCase LLC` from now on; no longer `Armedia LLC`.

Reply:
updated that